packages/lesswrong/server/authenticationMiddlewares.ts

```diff
@@ -178,14 +232,47 @@ export const addAuthMiddlewares = (addConnectHandler) => {
     })
   }
   
+
+  // NB: You must also set the expressSessionSecret setting
+  const auth0ClientId = auth0ClientIdSetting.get();
+  const auth0OAuthSecret = auth0OAuthSecretSetting.get()
+  const auth0Domain = auth0DomainSetting.get()
+  if (auth0ClientId && auth0OAuthSecret && auth0Domain) {
+    passport.use(new Auth0Strategy(
+      {
+        clientID: auth0ClientId,
+        clientSecret: auth0OAuthSecret,
+        domain: auth0Domain,
+        callbackURL: `${getSiteUrl()}auth/auth0/callback`,
+      },
+      createAuth0UserHandler('services.auth0.id', profile => profile.id, async profile => {
+        delete profile._json
```

Comment:
Might be nice to have a comment explaining why?

Reply:
Could also use something like lodash `omit`, I have a vague taste sense that deleting object properties is sort of poor form, but given that you're immediately returning it doesn't seem like a big deal.